packages/components/src/base-field/styles.ts

```diff
@@ -13,7 +13,7 @@ export const BaseField = css`
 	background: ${ CONFIG.controlBackgroundColor };
 	border-radius: ${ CONFIG.controlBorderRadius };
 	border: 1px solid;
-	border-color: ${ CONFIG.controlBorderColor };
+	border-color: ${ COLORS.ui.border };
```

Comment:
Remove the `CONFIG.controlBorderColor*` variables and consolidate to `COLORS.ui`. (It was only used in this one instance.)